pkg/cli/mirror/cincinnati_graph_image.go

```diff
@@ -128,7 +125,7 @@ func (o *MirrorOptions) buildGraphImage(ctx context.Context, dstDir string) (ima
 }
 
 // downloadsGraphData will download the current Cincinnati graph data
-func downloadGraphData(ctx context.Context, dir, url string) error {
+func downloadGraphData(ctx context.Context, dir string) error {
```

Comment:
The `dir` input should be sanitized before use given it is a directory path which can contain plenty of random stuff. This was one of the few things I fixed in my prodsec PR. We should continue to keep the practice going.

Reply:
Good feedback. This is actually getting sanitized before it is opened/used. Line number below.

https://github.com/openshift/oc-mirror/blob/7bde4c7bde6b829a34592b2d12115fb38b15576f/pkg/cli/mirror/cincinnati_graph_image.go#L136

I did a go sec run through and made a few fixes though.